app/views/shareable_links/my_modules/_start_date_label.html.erb

```diff
@@ -0,0 +1,9 @@
+<span class="start-date-label">
+  <% if my_module.started_on.present? %>
+    <span class="iso-formatted-date font-bold"><%= l(my_module.started_on, format: :full_iso) %></span>
```

Comment:
I think you can simply use my_module.started_on.iso8601

Reply:
Yes that's right, but we will have to handle it differently in JS